discord/webhook.py

```diff
@@ -400,6 +401,16 @@ async def foo():
         webhook = Webhook.partial(123456, 'abcdefg', adapter=RequestsWebhookAdapter())
         webhook.send('Hello World', username='Foo')
 
+    .. container:: operations
+        .. describe:: x == y
+            Checks if two webhooks are equal.
+        .. describe:: x != y
+            Checks if two webhooks are not equal.
+        .. describe:: hash(x)
+            Returns the webhooks's hash.
+        .. describe:: str(x)
+            Returns the webhooks's name.
```

Comment:
These usually have newlines between them for rendering purposes. Take a look at the `Member` docstring for how it's done.